pkg/internal/testhelpers/containers.go

```diff
@@ -99,6 +111,14 @@ func (e ExtensionState) UsesTimescale2() bool {
 	return (e & timescale2Bit) != 0
 }
 
+func (e ExtensionState) UsesTimescaleNighty() bool {
+	return (e & timescaleNightyBit) != 0
+}
+
+func (e ExtensionState) UsesTimescaleNightyMultinode() bool {
```

Comment:
This is never used and I dont think we want combo checks like this. The same can be done with UsesTimescaleNighty() && UsesMultinode()